db/comdb2.h

```diff
@@ -1252,18 +1253,14 @@ struct ireq {
     /* REGION 1 */
     /************/
     char *frommach; /* machine that the buffer came from */
```

Comment:
If the purpose is to reduce `sizeof(struct ireq)`, may I propose to re-order region 1 as well.